src/screens/AddNotificationEmailScreen.tsx

```diff
@@ -118,9 +118,10 @@ class AddNotificationEmailScreen extends PureComponent<Props, State> {
   };
 
   render() {
-    const { email, error } = this.state;
+    const { email } = this.state;
+    const { error, route, isLoading } = this.props;
```

Comment:
I don't know what to gain from it. If there is any error left in the store after some previous action, the user will see it when going to this screen.

Reply:
gain is that we don't have two sources of truth, I can reset error on a component did mount to be sure that there is no error

Reply:
added